src/scripts/DataLayerListenerManagerFactory.js

```diff
@@ -136,116 +140,69 @@ ListenerManagerFactory.create = function(dataLayerManager) {
   /**
    * Checks if the listener matches the item.
    *
-   * @param {Item} listener The listener.
+   * @param {Listener} listener The listener.
    * @param {Item} item The item.
    * @returns {Boolean} true if listener matches the item, false otherwise.
    * @private
    */
-  function _isMatching(listener, item) {
-    const listenerConfig = listener.config;
+  function _matches(listener, item) {
+    const event = listener.event;
     const itemConfig = item.config;
-    let isMatching = false;
+    let matches = false;
 
     if (item.type === constants.itemType.DATA) {
-      if (listenerConfig.on === constants.dataLayerEvent.CHANGE) {
-        isMatching = _isSelectorMatching(listenerConfig, item);
+      if (event === constants.dataLayerEvent.CHANGE) {
+        matches = _selectorMatches(listener, item);
       }
     } else if (item.type === constants.itemType.EVENT) {
-      if (listenerConfig.on === constants.dataLayerEvent.EVENT ||
-        listenerConfig.on === itemConfig.event) {
-        isMatching = _isSelectorMatching(listenerConfig, item);
+      if (event === constants.dataLayerEvent.EVENT || event === itemConfig.event) {
+        matches = _selectorMatches(listener, item);
       }
-      if (itemConfig.data &&
-        listenerConfig.on === constants.dataLayerEvent.CHANGE) {
-        isMatching = _isSelectorMatching(listenerConfig, item);
+      if (itemConfig.data && event === constants.dataLayerEvent.CHANGE) {
+        matches = _selectorMatches(listener, item);
       }
     }
-    return isMatching;
-  }
 
-  /**
-   * Returns the indexes of the registered listeners that match the listener off.
-   *
-   * @param {Item} listenerOff The listener off.
-   * @returns {Array} The indexes of the matching listeners.
-   * @private
-   */
-  function _getListenersMatchingListenerOff(listenerOff) {
```

Comment:
This method had a purpose. Did you completely remove this method or did you implement the logic differently?